packages/core/src/navigation/navigation-start.element.ts

```diff
@@ -120,36 +123,70 @@ export class CdsNavigationStart extends LitElement implements FocusableItem {
 
   private handleStartButtonText() {
     spanWrapper(this.childNodes);
+
     // get the projected text now wrapped in a span and add the sr attribute.
     this.querySelector('span')?.setAttribute(NAVIGATION_TEXT_WRAPPER, '');
```

Comment:
What happens if the button has a span tag inside of it instead of just text? Should we extend `spanWrapper` so we can add a classname or other identifier to it?

Reply:
The text will come from here:
```
<cds-navigation-start>Root or Group Text</cds-navigation-start>
```

Could they put other elements in there? yeah. I had this conversation early on when I implemented the span wrapping for sr text with one or two of us and came to the conclusion that while they shouldn't, for now, if it needs it I should write that up in the documentation. If folks start adding lots of other elements in there, then I can see if it should be extended to handle other things. Or, just be firm and explicit that folks should only be projecting text or icons or the override icon for the start button.  

wrt  adding a classname or identifier to `spanWrapper`, thats what  the attribute  `NAVIGATION_TEXT_WRAPPER` (cds-navigation-sr-text) is for it acts as a marker for the querySlot so that starts and items can find and update the `display:screen-reader-only` utility token as needed for the projected text. I ended up with that solution after initial PR feedback. I was initially doing just that with a spanWrapper fn that could take an optional class argument and it would get applied to all the spans that get wrapped. Iir the feedback request to separate the classname out was to keep the spanWrapper fn focused on only its wrapping job.